api/proto/disperser/disperser.proto

```diff
@@ -22,6 +28,11 @@ service Disperser {
 	// 4. The Disperser verifies the signature and returns a DisperseBlobReply message.
 	rpc DisperseBlobAuthenticated(stream AuthenticatedRequest) returns (stream AuthenticatedReply);
 
+	// DispersePaidBlobAuthenticated is similar to DispersePaidBlob, except that it requires the
+	// client to authenticate itself via the AuthenticationData message. The protoco is similar to 
+	// DisperseBlobAuthenticated.
+	rpc DispersePaidBlobAuthenticated(stream AuthenticatedPaidRequest) returns (stream AuthenticatedReply);
```

Comment:
Not sure why we'd need another endpoint. `DispersePaidBlob` already has authenticated built in and has a simpler interface. I feel like having 4 different dispersal endpoints would be very confusing

Reply:
I don't think `DispersePaidBlob` has authentication built in since we removed signature for PaymentHeader. Perhaps we don't want a `DispersePaidBlob` but we do want `DispersePaidBlobAuthenticated`?

Reply:
Hmm what if we add `signature` as a top level field in `DispersePaidBlobRequest`, which would sign over the payment header? 
@mooselumph any thoughts?

Reply:
I would agree that `DispersePaidBlobAuthenticated` and `DispersePaidBlobRequest` don't need to exist at the same time and payment should be authenticated regardless of what the method name is called